caravel/daemons/migration.py

```diff
@@ -16,8 +17,12 @@ def grouper(iterable, n, fillvalue=None):
 
 @app.route("/_internal/migrate_schema")
 def migrate_schema():
+    horizon = datetime.datetime.now() - model.Listing.MARK_AS_OLD_AFTER
```

Comment:
This migration will cause a lot of things to disappear. Would it be useful to send an email to the seller or post an alert on the landing page?